lang/ckb/lang.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * @license    GPL 2 (http://www.gnu.org/licenses/gpl.html)
+ *
+ * @author qezwan <qezwan@gmail.com>
+ */
+$lang['menu']                  = 'پێڕستی خزمەتگوزاری';
+$lang['fetch']                 = 'نیشاندان';
+$lang['install']               = 'دامەزراندن';
+$lang['delete']                = 'سڕینەوە';
+$lang['check']                 = 'پشکنین';
+$lang['no_repos']              = 'هیچ بەستەری بارکراو شێوەبەند نەکراوە.';
+$lang['disabled']              = 'ناچالاککراوە';
+$lang['conn_err']              = 'هەڵەی پەیوەندی';
+$lang['dir_err']               = 'ناتوانێت فۆڵدەری کاتی دروست بکات بۆ وەرگرتنی ڕووکار';
+$lang['down_err']              = 'ناتوانێت ڕووکار وەربگرێت';
+$lang['zip_err']               = 'هەڵەی دروست کردن یان دەرهێنانی زیپ';
+$lang['install_ok']            = 'ڕووکار بە سەرکەوتوویی دامەزرا. ڕووکارە نوێیەکە لە شریتی ئامڕازی لاپەڕەی بژارکردن یان لەگەڵ <code>js#theme_name</code>.';
+$lang['install_no']            = 'هەڵەی پەیوەندی. هەرچۆنێک بێت دەتوانیت ڕووکارەکەت بە دەستی باربکەیت لە <a href="http://samuele.netsons.org/dokuwiki/lib/plugins/indexmenu/upload/">لێرە</a>.';
+$lang['delete_ok']             = 'ڕووکار بە سەرکەوتوویی سڕاوەتەوە.';
+$lang['delete_no']             = 'هەڵەیەک ڕوویدا لەکاتی سڕینەوەی ڕووکار';
+$lang['upload']                = 'هاوبەش کردن';
+$lang['checkupdates']          = 'نوێکردنەوەکانی پێوەکراو';
+$lang['noupdates']             = 'پێڕستی پێڕست پێویستی بە نوێکردنەوە نییە. تۆ پێشتر دوا وەشانت هەیە:';
+$lang['infos']                 = 'دەتوانیت ڕووکارەکەت دروست بکەیت دوای ڕێنماییەکان لە <a href="https://www.dokuwiki.org/plugin:indexmenu#theme_tutorial">Theme Tutorial</a> page. <br/>دواتر دەتوانیت زیاتر خەڵک دڵخۆش بکەیت :-) ناردنی بۆ کۆگای پێرستی گشتی، لەگەڵ دوگمەی "هاوبەشکردن" لەژێر ئەو ڕووکارە.';
+$lang['showsort']              = 'ژمارەی ڕیزکردنی پێڕستی پێڕست: ';
+$lang['donation_text']         = 'پێوەکراوەی پێرستمی هیچ کەسێک سپۆنسەری ناکات بەڵام من بە خۆڕایی لە کاتی یەدەگی خۆمدا پەرەی پێ دەدەم و پشتگیری دەکەم . ئەگەر شتێکی سوپاسگوزارت بۆ بەدەست بهێنی یان دەتەوێت پشتگیری لە گەشەی بکەیت، دەتوانیت بیر لە بەخشینێک بکەیت.';
+$lang['js']['indexmenuwizard'] = 'ویزاردی پێرست';
+$lang['js']['index']           = 'پێڕست';
+$lang['js']['options']         = 'هەڵبژاردنەکان';
+$lang['js']['navigation']      = 'ڕێنیشاندەر';
+$lang['js']['sort']            = 'پۆلێنکردن';
+$lang['js']['filter']          = 'فلتەر';
+$lang['js']['performance']     = 'ئەدا';
+$lang['js']['namespace']       = 'بۆشایی ناو';
+$lang['js']['nsdepth']         = 'قوڵایی';
+$lang['js']['js']              = 'درەخت لەلایەن Javasript ەوە بە چاپ درا، دەتوانیت ڕووکاری خۆت پێناسە کەیت';
```

Comment:
@qezwan i think a `c` is lost in javascript. In the Translation tool you can also submit changes.(it takes some time before merged changes here show up in the Translation Tool as well. It is practical to submit changes after changes synced.)